app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -178,7 +199,14 @@ public void onResume()
         preview.enableView();
     }
 
-    private void storeStudentImages(){
+    private synchronized void storeStudentImages(){
+        Student student = new Student();
+        studentDao.insert(student);
+        StudentImageCollectionEvent studentImageCollectionEvent = new StudentImageCollectionEvent();
+        studentImageCollectionEvent.setStudent(student);
```

Comment:
Until we have successfully extracted features for a Student, the "student" property in a StudentImageCollectionEvent should probably be left as `null`, right?